README.md

```diff
@@ -154,7 +201,7 @@ How many valid sentences are there in this language?
 
 ## A2.6
 
-«replace this with your answer»
+Assuming you CAN repeat adjectives, there are in infinite number of valid sentences because you can infinitely repeat adjectives. Assumming you CANNOT repeat adjectives, there are 60 valid sentences.
```

Comment:
# 3 

# +1 bonus for the CYA explanation, which is just what is required in the real world....